modules/user_manual/pages/online_collaboration.adoc

```diff
@@ -0,0 +1,41 @@
+= Online Collaboration
+:collabora-online-url: https://www.collaboraoffice.com/collabora-online/
+:libreoffice-url: https://www.libreoffice.org/
+
+== Collabora Online
+
+{collabora-online-url}[Collabora Online] is a powerful {libreoffice-url}[LibreOffice]-based online office that supports all major document, spreadsheet and presentation file formats, and is integrable with ownCloud. 
+
+=== Secure View
+
+Secure View is a way to securely distribute information, without that information being able to leave your organisation.
+It provides the ability to restrict access to documents, in _read-only_ shares, so that recipients cannot copy, download, or edit shared files.
+
+When enabled, documents will be viewed through Collabora Online with a watermark.
+In addition, the ability to print and export files (with watermarks) is available, once Secure View is enabled.
+
+NOTE: This feature is *not* available in the community edition.
+
+==== How to Use Secure View
+
+image::enterprise/collaboration/secure-view/collabora-online-administration.png[Configure Secure View in ownCloud Server.]
+
+To enable Secure View for an existing, or new, share:
+
+* Disable btn:[can edit]; Users will only be able to view documents.
```

Comment:
To add: `Secure View (with watermarks)` is mutually exclusive with `can edit`. Enabling one of them will disable the other.